src/kuzzle.js

```diff
@@ -428,6 +428,32 @@ Kuzzle.prototype.logout = function (cb) {
 };
 
 /**
+ * Checks wether a given jwt token still represents a valid session in Kuzzle.
+ *
+ * @param  {string}   token     The jwt token to check
+ * @param  {function} callback  The callback to be called when the response is
+ *                              available. The signature is `function(error, response)`.
+ * @return {Kuzzle}             The Kuzzle instance to enable chaining.
+ */
+Kuzzle.prototype.checkToken = function (token, callback) {
```

Comment:
Should be added to the list of functions to be promisified (at the end of the constructor function)